site/testsuites/testsuite-javascript/__tests__/tbdex/pfi/creatingQuotes.test.js

```diff
@@ -123,15 +123,15 @@ describe('PFI: Quotes', () => {
           amount: '0.01',
           fee: '0.0001',
           paymentInstruction : {
-            link: 'https://example.com',
+            link: 'https://example.pfi.io/instructions',
```

Comment:
```suggestion
            link: 'https://example.com/instructions',
```
let's keep example.com as it's a default domain anyone can legally use in copy

Reply:
need to update the JS links too